### PR TITLE
docs for v1.4.0 release

### PR DESCRIPTION
**Pull request by KeefeL**
## Description

Add release docs for the upcoming v1.4.0

## Rationale

NA

### CODE REVIEW

File: docs/release-notes/releaseNotes.md [hunk 1]
```diff
@@ -7,6 +7,16 @@ order: 7
 ---
 
 # Release Notes
+## Greenfield v1.4.0 - Ural Upgrade
+BNB Greenfield v1.4.0 introduces the Ural Hardfork, which includes the following features:
+1. Allow other accounts with permission to set resource tags, not just the owner.
```
Comment (zjubfd) [inline docs/release-notes/releaseNotes.md:12]: please try to improve the writting here.

File: docs/release-notes/releaseNotes.md [hunk 2]
```diff
@@ -7,6 +7,16 @@ order: 7
...
+
+2. Support bucket migration: In Greenfield, each bucket is associated with a primary SP. If a user is dissatisfied with
```
Comment (zjubfd) [inline docs/release-notes/releaseNotes.md:14]: Another one is the permission hub? cross chain enhancement?

Comment (KeefeL) [reply docs/release-notes/releaseNotes.md:14]: Permission hub is already supported in v1.3, not 1.4

File: docs/release-notes/releaseNotes.md (hunk 1 above)
Comment (KeefeL) [reply docs/release-notes/releaseNotes.md]: fixed

**Review by zjubfd — APPROVED** [review]

**Pull request merged**